catalogs/hadoop-common/src/main/java/org/apache/gravitino/catalog/hadoop/fs/HDFSFileSystemProvider.java

```diff
@@ -48,4 +55,29 @@ public String scheme() {
   public String name() {
     return BUILTIN_HDFS_FS_PROVIDER;
   }
+
+  /**
+   * Add additional HDFS specific configurations.
+   *
+   * @param configs Original configurations.
+   * @return Configurations with additional HDFS specific configurations.
+   */
+  private Map<String, String> additionalHDFSConfig(Map<String, String> configs) {
+    Map<String, String> additionalConfigs = Maps.newHashMap(configs);
+
+    // Avoid multiple retries to speed up failure in test cases.
+    // Use hard code instead of CommonConfigurationKeysPublic.IPC_CLIENT_CONNECT_TIMEOUT_KEY to
+    // avoid dependency on a specific Hadoop version.
+    if (!configs.containsKey(HDFS_IPC_CLIENT_CONNECT_TIMEOUT_KEY)) {
+      additionalConfigs.put(HDFS_IPC_CLIENT_CONNECT_TIMEOUT_KEY, DEFAULT_CONNECTION_TIMEOUT);
+    }
+
+    if (!configs.containsKey(HDFS_IPC_PING_KEY)) {
+      additionalConfigs.put(HDFS_IPC_PING_KEY, "true");
```

Comment:
I found that you have several customized conf values here and above, you'd also define constants for these.